oxidauth-usecases/src/bootstrap/mod.rs

```diff
@@ -362,6 +391,8 @@ async fn first_or_create_authority(
                     let authority_settings = AuthoritySettings {
                         jwt_ttl: DEFAULT_JWT_TTL,
                         refresh_token_ttl: DEFAULT_REFRESH_TOKEN_TTL,
+                        require_2fa: DEFAULT_2FA,
+                        totp_jwt_ttl: DEFAULT_TOTP_TOKEN_TTL,
```

Comment:
This should be an Option of totp_jwt_ttl - edit code to work with that and set default to none